docs/basics.md

```diff
@@ -254,7 +254,7 @@ You can customize priority by frontend. The priority value is added to the rule
       rule = "PathPrefix:/toto"
 ```
 
-Here, `frontend1` will be matched before `frontend2` (`(3 + 10 == 13) > (4 + 5 == 9)`).
+Here, `frontend1` will be matched before `frontend2` (`(14 + 10 == 24) > (16 + 5 == 21)`).
```

Comment:
I think that in reality, if you specify a priority, you don't use the length and use directly the value. Can you verify this ?

Reply:
In Traefik 1.5, a custom priority _does_ ignore the length-based factor according to my tests not too long ago.

Reply:
Yes, I made a mistake, the priority override the rule length according to https://github.com/containous/traefik/blob/master/server/server.go#L1192